inspirehep/modules/records/jsonschemas/records/jobs.json

```diff
@@ -22,21 +22,17 @@
             "uniqueItems": true
         },
         "continent": {
-            "items": {
-                "enum": [
-                    "North America",
-                    "Europe",
-                    "Asia",
-                    "South America",
-                    "Africa",
-                    "Oceania"
-                ],
-                "title": "Continent",
-                "type": "string"
-            },
-            "title": "Continent",
-            "type": "array",
-            "uniqueItems": true
+            "type": "string",
+            "enum": [
+                "North America",
+                "Europe",
+                "Asia",
+                "South America",
+                "Australasia",
+                "Middle East",
+                "Africa",
+                "Europe / South America"
```

Comment:
Oh, right, I cut/pasted the values from [here](https://github.com/inspirehep/inspire-next/blob/fec36228b9b06a1d2da18c2df825558f323fbc0a/inspirehep/dojson/current_marcxml_usage/jobs.txt#L45-L65), but I guess it's better to have a correct `enum` here and fix the data on Legacy.